src/Security/MemberAuthenticator/CMSAuthenticator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace SilverStripe\Security\MemberAuthenticator;
+
+use SilverStripe\Security\Authenticator as BaseAuthenticator;
+use SilverStripe\Security\Member;
+
+class CMSAuthenticator extends Authenticator
+{
+
+    public function supportedServices()
+    {
+        return BaseAuthenticator::CMS_LOGIN;
+    }
+
+    /**
+     * @param array $data
+     * @param $message
+     * @param bool $success
+     * @return Member
+     */
+    protected function authenticateMember($data, &$message, &$success, $member = null)
```

Comment:
4th parameter made it more extensible to just call authenticateMember, instead of having to go through authenticate itself.